java/daikon/chicory/DeclWriter.java

```diff
@@ -714,6 +480,115 @@ private String out_name(Enum<?> e) {
     return e.name().toLowerCase();
   }
 
+  /**
+   * Output most of the decl file information for a single variable.
```

Comment:
Regarding "most", what information is not output?

Also, in what the format is the information output?  Is it output to stdout, or to somewhere else?

Reply:
I have updated the comment header:
    * Output most of the decl file information for a single variable.
   * This includes the variable, var-kind, enclosing-var, array, dec-type, rep-type, constant, function-args, flags, and
   * comparability records. Most notably, it does not output the parent record.  The records are output via the PrinterWriter
   * passed as an argument to the DeclWriter constructor.